lib/addins/WindowCoveringTilt.js

```diff
@@ -0,0 +1,209 @@
+/* Sample module - Simple handler for rolling shutter actuator  
+ * 
+ */
+'use strict';
+
+/**
+ * @type {HandlerPattern}
+ */
+var HandlerPattern = require('./handlerpattern.js');
+var log = require('debug')('WindowCoveringTilt');
+
+/**
+ * @class A custom handler for the "Jalousie Aktor" (rolling shutter/blinds actuator) - including Tilt
+ * @extends HandlerPattern
+ */
+class WindowCoveringTilt extends HandlerPattern {
+
+	/*******************************************************************************************************************
+	 * onKNXValueChange is invoked if a Bus value for one of the bound addresses is received
+	 * 
+	 */
+	onKNXValueChange(field, oldValue, knxValue) {
+		// value for HomeKit
+		var newValue;
+		log('INFO: onKNXValueChange(' + field + ", "+ oldValue + ", "+ knxValue+ ")");
+
+		if (field === "TargetPosition") {
+			// TargetPosition is DPT5.001 Percentage (0..255)
+			// need to convert to (0..100) first
+			// Homekit is using %-open, meaning 0% is closed/down
+			this.lastCommand = 'target';
+			
+			newValue = 100 - knxValue*100/255;
+			
+			if (newValue > this.myAPI.getValue("CurrentPosition")) {
+				// newValue is higher, shutter's moving up
+				this.myAPI.setValue("PositionState", 1);
+
+			} else if (newValue < this.myAPI.getValue("CurrentPosition")){
+				// newValue is higher, shutter's moving down
+				this.myAPI.setValue("PositionState", 0);
+
+			}
+			
+			this.myAPI.setValue("TargetPosition", newValue);
+
+		} else if (field === "CurrentPosition") {
+			// Current Position is sent by the actuator if the Movement has stopped a new postion is reached
+			
+			// CurrentPosition is DPT5.001 Percentage (0..255)
+			// need to convert to (0..100) first
+			// Homekit is using %-open, meaning 0% is closed/down
+			newValue = 100 - knxValue*100 / 255;
+			
+			this.myAPI.setValue("CurrentPosition", newValue); // inform homekit
+			this.myAPI.setValue("PositionState", 2); //stopped
+
+
+			// return to stopped immediately, and set the Target to Current
+			this.myAPI.setValue("TargetPosition", this.myAPI.getValue("CurrentPosition"));
+
+		} else if (field==="ShutterMove") {
+			// this isn't a characteristic, we need this extra object to catch switch use, too
+			// The actuator is lowering the rolling shutters if a 1 is received, and 
+			// raises on a 0
+
+			switch (knxValue) {
+				case 0:
+					this.myAPI.setValue("TargetPosition", 100); // top position, so home shows "opening"
+					this.myAPI.setValue("PositionState", 0); //up
+					break;
+
+				case 1:
+					this.myAPI.setValue("TargetPosition", 0); // low position, so home shows "closing"
+					this.myAPI.setValue("PositionState", 1); //down
+					break;
+
+			} // switch
+
+		} else if (field === "TargetHorizontalTiltAngle") {
+			// TargetPosition is DPT5.001 Percentage (0..255)
+			// need to convert to (0..100) first
+			// Homekit is using %-open, meaning 0% is closed/down
+			
+			newValue = 100 - knxValue*100 / 255;
```

Comment:
According to Docu the HomeKit value is supposed [to be an angle from -90° to 90°](https://github.com/KhaosT/HAP-NodeJS/blob/master/lib/gen/HomeKitTypes.js#L1832)  
Can you make sure that it is *not* an angle but a percentage?  
Same for Current- and of course reverse conversion from HomeKit to KNX